### PR TITLE
Resolve #2

### LINKED ISSUE #2: Support for the Dim decoration
Taking this spec:
https://misc.flogisoft.com/bash/tip_colors_and_formatting

There is also a "Dim" mode: `\e[2mDim`.

It would be nice to have that available in this library. I'd file a PR if this is up for grabs.

### PR DESCRIPTION
**Pull request by nikeee**

### CODE REVIEW

File: Crayon/Decorations.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@ namespace Crayon
     internal struct Decorations
```
Comment (nikeee) [inline Crayon/Decorations.cs:3]: Why not use an enum here btw?

Comment (riezebosch) [reply Crayon/Decorations.cs:3]: If (I recall correctly)  then you have to explicitly cast it to an int to get the underlying value in the string conversion so it didn't make much sense to use an enum altogether.

File: Crayon.Console/Program.cs [hunk 2]
```diff
@@ -11,12 +11,12 @@ static void Main(string[] args)
             Console.WriteLine("normal");
             Console.WriteLine(Output.BrightBlue($"Bright {Output.Green("and normal green")}"));
 
-            Console.WriteLine(Output.Green($"The difference {Output.Bold("between bold")} and {Output.BrightGreen("bright green")}"));
-            
+            Console.WriteLine(Output.Green($"The difference {Output.Bold("between bold")}, {Output.BrightGreen("bright green")} and {Output.Dim("dim")}"));
```
Comment (riezebosch) [inline Crayon.Console/Program.cs:14]: I don't see much difference between normal and dim on my screen but I believe you.

**Comment by nikeee** [comment]
It looks like this (Terminator, similar in other emulators):
image

**Pull request merged**